ui/packages/catalog-ui-search/src/main/webapp/js/filter.js

```diff
@@ -332,6 +332,8 @@ function matchesFilter(metacard, filter) {
                         if (matchesCIRCLE(valuesToCheck[i], filter)) {
                             return true;
                         }
+                    } else if (CQLUtils.isPolygonFilter(filter)) {
```

Comment:
@andrewkfiedler Where/how does this get called (from the UI interaction perspective) so I can verify if this is all that needs to be done?

Reply:
@andrewkfiedler filter added. Should work as expected now.